interop-tests/src/test/scala/akka/grpc/scaladsl/RichErrorModelSpec.scala

```diff
@@ -84,9 +84,12 @@ class RichErrorModelSpec
         GreeterServiceHandler(RichErrorImpl, eHandler = (_: ActorSystem) => customHandler)
       // #custom_eHandler
 
-      Http(system).newServerAt(interface = "127.0.0.1", port = 8080).bind(service)
+      val bound = Await.result(
```

Comment:
Since this tests imports `with ScalaFutures` you can also use `.futureValue`, but `Await.result` works fine too :+1: